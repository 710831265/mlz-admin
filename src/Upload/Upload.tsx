import React, { useState, useContext } from 'react';
import { Upload as AntdUpload } from 'antd';

const Upload = AntdUpload;

export default Upload;
