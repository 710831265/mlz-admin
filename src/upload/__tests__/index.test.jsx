import React, { Component } from 'react';
import testMount from '../../../tests/testMount';
import testSnapshot from '../../../tests/testSnapshot';
import Upload from '..';

describe('ðŸ“¦  Upload', () => {
  testMount(Upload);
  testSnapshot(Upload);
});
