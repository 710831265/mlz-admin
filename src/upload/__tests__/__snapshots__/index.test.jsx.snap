// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ðŸ“¦  Upload snapshot test snapshot comparing 1`] = `"<span class><div class=\\"ant-upload ant-upload-select ant-upload-select-text\\" style=\\"display: none;\\"><span tabindex=\\"0\\" class=\\"ant-upload\\" role=\\"button\\"><input type=\\"file\\" style=\\"display: none;\\" accept></span></div><div class=\\"ant-upload-list ant-upload-list-text\\"></div></span>"`;
